examples/asr/asr_adapters/train_asr_adapter.py

```diff
@@ -134,11 +141,11 @@ def main(cfg):
 
     # Setup model for finetuning (train and validation only)
     cfg.model.train_ds = update_model_cfg(model.cfg.train_ds, cfg.model.train_ds)
-    cfg.model.validation_ds = update_model_cfg(model.cfg.validation_ds, cfg.model.validation_ds)
-
-    # Call the dataloaders and optimizer + scheduler
     model.setup_training_data(cfg.model.train_ds)
-    model.setup_multiple_validation_data(cfg.model.validation_ds)
+
```

Comment:
Is it possible to init an ASR model without train_ds?

Reply:
No, this is a train script, at least train_ds must be provided. Otherwise no point calling this script.